frontend/src/views/ManureAndImports/ManureAndImports.tsx

```diff
@@ -54,6 +54,22 @@ export default function ManureAndImports() {
   const [manureFormData, setManureFormData] =
     useState<NMPFileImportedManureData>(DefaultManureFormData);
 
+  const [tab, setTab] = useState<Array<string>>([
```

Comment:
SUPER minor but could these names be changed to [tabs, setTabs] just to make it clear this is an array?